mappings/servers/slaze/metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "slaze",
+    "name": "Slaze",
+    "description": "Slaze is a brazilian server formed by passionate players!",

```

Comment:
The description needs to be between 16-50 characters long (as per the [docs](https://docs.cheatbreaker.net/server-mappings/adding-servers/metadata#basic))

Yours is 57.